docs/rules/new-cap.md

```diff
@@ -8,77 +8,135 @@ var friend = new Person();
 
 ## Rule Details
 
-This rule is aimed at helping to distinguish regular functions from constructor functions. As such, it warns whenever it sees `new` followed by an identifier that isn't capitalized or whenever it sees capitalized function called directly without `new` operator.
+This rule requires constructor names to begin with a capital letter. Certain built-in identifiers are exempt from this rule. These identifiers are:
 
-The following patterns are considered problems:
+* `Array`
+* `Boolean`
+* `Date`
+* `Error`
+* `Function`
+* `Number`
+* `Object`
+* `RegExp`
+* `String`
+* `Symbol`
+
+## Options
+
+This rule has an object option:
+
+* `"newIsCap": true` (default) requires all `new` operators to be called with uppercase-started functions.
+* `"newIsCap": false` disables checks for all `new` operators to be called with uppercase-started functions.
+* `"capIsNew": true` (default) requires all uppercase-started functions to be called with `new` operator.
+* `"capIsNew": false` disables checks for all uppercase-started functions to be called with `new` operator.
+* `"newIsCapExceptions"` allows specified lowercase-started function names to be used with the `new` operator.
+* `"capIsNewExceptions"` allows specified uppercase-started function names to be used without the `new` operator.
```

Comment:
I thought of that, but it seemed really messy. Plus, these are allowed whether or not capIsNewExceptions is used. They are built-ins rather than defaults.